src/SprykerEco/Service/ComputopApi/ComputopApiServiceInterface.php

```diff
@@ -40,6 +40,7 @@ public function generateEncryptedMac(ComputopApiRequestTransfer $requestTransfer
     /**
      * Specification:
      * - Generate header transfer by response array
+     * - Requires `ComputopApiResponseHeaderTransfer.mac` and `ComputopApiResponseHeaderTransfer.method` to bet set.
```

Comment:
not required
the only input parameters are
```
 @param array $plaintextResponseHeader
 @param string $method
 ```

Reply:
Fixed